src/apis/AppState/index.js

```diff
@@ -2,7 +2,7 @@ import ExecutionEnvironment from 'fbjs/lib/ExecutionEnvironment';
 import findIndex from 'array-find-index';
 import invariant from 'fbjs/lib/invariant';
 
-const EVENT_TYPES = [ 'change' ];
+const EVENT_TYPES = ['change'];
```

Comment:
Actually, I've had to go back and forth multiple times depending on the company and project. I'm just tired of switching :)